src/bank.rs

```diff
@@ -248,7 +248,7 @@ impl Module for BankKeeper {
                 res.amount = amount;
                 Ok(to_json_binary(&res)?)
             }
-            q => bail!("Unsupported bank query: {:?}", q),
+            _ => unimplemented!("bank query: {request:?}",),
```

Comment:
Same as above, this is exactly what I wanted to achieve, in the case of the bank module, all messages and queries must be implemented in MultiTest and the `unimplemented!()` part should never be executed. In case of queries there are two additional queries to be implemented as soon as possible: `DenomMetadata` and `AllDenomMetadata`. In case someone uses message or query without required feature enabled from cosmwasm-std - test will panick.